test/spec/mapbox_logo.js

```diff
@@ -17,11 +17,17 @@ describe('mapbox_logo', function() {
     });
 
     it('is not on tilejson map without mapbox_logo flag', function() {
-        var map = L.mapbox.map(element, tileJSON);
+        var map = L.mapbox.map(element, helpers.tileJSON_mapboxlogoMissing);
         var mapboxLogoControl = map._mapboxLogoControl.getContainer();
         expect(L.DomUtil.hasClass(mapboxLogoControl, 'mapbox-logo-true')).to.be(false);
     });
 
+    it('is on map when tilejson is mapbox styleJSON without mapbox_logo flag', function() {
+        var map = L.mapbox.map(element, helpers.styleJSON);
```

Comment:
`helpers.styleJSON` has `"mapbox_logo": true`, so either the title of this test needs to change or the `mapbox_logo` key needs to be removed from `helpers.styleJSON`.

Reply:
oh, blorp - yeah. taken care of in 5aec34ac